server-nodejs/extensions/jwt_delegate.js

```diff
@@ -15,9 +15,15 @@ module.exports = function (app) {
                     console.warn("Connection with the server : ", damas.server, " impossible.");
                     return next();
                 }
-                db.create([user.user], function (err, nodes) {
+                let userNode;
+                if (!user.user) {
+                    userNode = user;
```

Comment:
is backward compatibility possible? since the password is not set in the response message in older versions